doc/nrf/links.txt

```diff
@@ -110,6 +110,8 @@
 
 .. _`SMP over Bluetooth`: https://github.com/apache/mynewt-mcumgr/blob/master/transport/smp-bluetooth.md
 
+.. _`breathe issue #437`: https://github.com/michaeljones/breathe/issues/437`
```

Comment:
Still a ` that shouldn't be there.

Reply:
Not my brightest hour, this PR... :) Fixed